gprofiler/profilers/python.py

```diff
@@ -242,7 +245,13 @@ def _profile_process(self, process: Process, duration: int, spawned: bool) -> Pr
 
     def _select_processes_to_profile(self) -> List[Process]:
         filtered_procs = []
-        for process in pgrep_maps(DETECTED_PYTHON_PROCESSES_REGEX):
+        all_processes = []
```

Comment:
```suggestion
```

Don't need to assign it, both sides of the `if` assign it

Reply:
Done